contents/customers/hostai.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How HostAI prevents churn with PostHog and LangFuse
+customer: HostAI
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1715246033/posthog.com/contents/host-ai-logo.jpg
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1715246033/posthog.com/contents/host-ai-logo.jpg
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/v1715246033/posthog.com/contents/hostai-screenshot.png
+industries:
+  - AI
+users:
+  - Engineering
+  - Leadership
+  - Founder
+toolsUsed:
+  - Feature flags
+  - Product analytics
+date: 2024-05-09
+---
+
+[HostAI](https://hostai.app/) is an AI platform that handles repetitive tasks for vacation home managers. Leveraging LLMs, a core feature is automating responses to guest inquiries.
+
+"The effectiveness of our platform hinges on the precision of our LLM responses. It’s not just about automating replies; it’s about ensuring each response is contextually relevant and exceptionally accurate." explains Punn Kam, co-founder of HostAI.
+
+## Identifying bad responses with PostHog and LangFuse
+
+HostAI tracks the quality of their LLM responses with [Langfuse](https://langfuse.com/). Then, using [PostHog's Langfuse integration](/docs/product-analytics/llms#langfuse), they combine this data with their [product analytics](/product-analytics). This enables them to pinpoint poor LLM responses and debug the root cause of them.
+
+"We immediately investigate if a customer's evaluation score begins to decrease and search their PostHog events for frequent message edits and rejections. Once we've found the problematic experience, we dig into their Langfuse traces to understand what went wrong. This approach has enabled us to increase our average evaluation score by 50%."
```

Comment:
Great story here, may be worth moving the 50% figure to the headline of the article -- always good to foreground a tangible result!